bundles/event_admin/event_admin_api/include/celix_event_constants.h

```diff
@@ -106,6 +106,47 @@ extern "C" {
  */
 #define CELIX_EVENT_TIMESTAMP "timestamp"
 
+/**
+ * @brief It is a property of event to indicate the event is a remote event. The type of the value for this event property is Boolean.
+ *
+ * If the value is true, the event will be delivered to remote
+ * event handlers and local event handlers, otherwise, the event will be only delivered to local event handlers.
+ */
+#define CELIX_EVENT_REMOTE_ENABLE "celix.event.remote.enable"
+
+/**
+ * @brief The QoS of the remote event. The type of the value for this event property is integer. It indicates the quality of service of the remote event. If the value is not set, the remote provider should use a proper default value.
+ *
+ * The value must be one of the following:
+ * - 0: At most once delivery
+ * - 1: At least once delivery
+ * - 2: Exactly once delivery
+ */
+#define CELIX_EVENT_REMOTE_QOS "celix.event.remote.qos"
```

Comment:
Should this value not impact whether a sendEvent return a success or not?

Reply:
No, this value does not affect whether sendEvent of event admin returns a success or not.
However, it will affect whether the sendEvent of the remote provider returns success or not. For example, if the MQTT client has not yet established a connection with the MQTT broker, it will immediately return ENOTCONN if the QOS value is 0. If the QOS value is greater than 0, it will wait until the connection is established before sending.

Reply:
Should QOS impact whether the `sendEvent` of event admin returns success or not? In my opinion, the remote provider and local event handler are in equivalent positions, so in the event admin, it only logs the error and does not return the error to the caller of the `sendEvent`.

Reply:
I agree that sound reasonable. Maybe good to mention this behaviour in the "celix.event.remote.qos" documentation.